Formula/chkrootkit.rb

```diff
@@ -1,9 +1,8 @@
 class Chkrootkit < Formula
   desc "Rootkit detector"
   homepage "http://www.chkrootkit.org/"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/c/chkrootkit/chkrootkit_0.50.orig.tar.gz"
```

Comment:
Squashed, sorry that took so long, I didn't have a current repo, on very slow internet in the country, and their are 304,746 items in the repo. I had been testing on my home tree and editing online.

I understand the need for guidelines, consistency, and the reduction in commits etc. But it also appears that it is possible to squash the commits of a pull request when merged by the maintainer. It's fairly new, not sure if it had been looked at as an option; eg: enforce consistency, in crontributing.md; but allow for minor commits to the pull request to by done the maintainers on merge to -master:
https://github.com/blog/2141-squash-your-commits

Hope this helps bump people it clears up a couple of false positives with ssh and adds some new test.
Best,
Ben